src/main/java/com/eafit/herzon/teis/models/CreditCard.java

```diff
@@ -0,0 +1,205 @@
+package com.eafit.herzon.teis.models;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.PrePersist;
+import jakarta.persistence.PreUpdate;
+import jakarta.persistence.Table;
+import jakarta.persistence.Temporal;
+import jakarta.persistence.TemporalType;
+import java.util.Date;
+
+/**
+ * Represents a credit card entity in the system.
+ */
+@Entity
+@Table(name = "credit_cards")
+public class CreditCard {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  @Column(name = "card_number", nullable = false, unique = true)
+  private String cardNumber;
+
+  @Column(nullable = false)
+  private String name;
+
+  @Column(name = "expiration_date", nullable = false)
+  private String expirationDate;
+
+  @ManyToOne
```

Comment:
I suggest to include the optional value to know whether relationships are obligatory or optional. In this case, the user relation can't be optional.
  ```suggestion
  @ManyToOne(optional = false)
  ```

Reply:
Fix: Ensure CreditCard always belongs to a User